src/pages/research_profile/ResearchProfile.tsx

```diff
@@ -85,67 +79,83 @@ const ResearchProfile = () => {
   ) : !person ? (
     <NotFound />
   ) : (
-    <BackgroundDiv>
-      <Helmet>
-        <title>{fullName}</title>
-      </Helmet>
-      <Typography variant="h1" sx={{ mt: '1rem', mb: '2rem' }}>
-        {fullName}
-      </Typography>
-      <Typography variant="h2">{t('common.employments')}</Typography>
-      {activeAffiliations.length > 0 ? (
-        <Box sx={lineSx}>
-          <WorkIcon />
-          <Box sx={textContainerSx}>
-            {activeAffiliations.map(({ organization }) => (
-              <AffiliationHierarchy key={organization} unitUri={organization} commaSeparated />
+    <div>
+      <Box sx={{ bgcolor: 'person.main', py: '1.1rem', display: 'flex', alignItems: 'center', gap: '1rem' }}>
+        <Typography variant="h1" sx={{ ml: '2rem' }}>
+          {fullName}
+        </Typography>
+        {orcidUri && <img src={orcidIcon} height="20" alt="orcid" />}
+      </Box>
+      <BackgroundDiv>
+        <Helmet>
+          <title>{fullName}</title>
+        </Helmet>
+
+        {activeAffiliations.length > 0 ? (
+          <Box sx={{ display: 'flex', gap: '0.5rem', mt: '1rem', flexDirection: { xs: 'column', lg: 'row' } }}>
+            {activeAffiliations.map(({ organization, role }) => (
+              <Box
+                key={organization}
+                sx={{
+                  width: 'fit-content',
+                  pr: '1.5rem',
+                  ':not(:last-child)': {
+                    borderRight: { xs: 'none', lg: '1px solid' },
+                    borderRightColor: 'primary.main',
+                    borderBottom: { xs: '1px solid', lg: 'none' },
+                    borderBottomColor: 'primary.main',
```

Comment:
kan forenkle dette litt også
```suggestion
                    borderBottom: { xs: '1px solid', lg: 'none' },
                    borderColor: 'primary.main',
```